conf/messages.en

```diff
@@ -400,14 +400,14 @@ isThisYourAddress.error.required = Select yes if this is your address
 
 indWhatIsYourPostcode.title = What is your postcode?
 indWhatIsYourPostcode.heading = What is your postcode?
-indWhatIsYourPostcode.hint = Enter your postcode to find your address automatically.
+indWhatIsYourPostcode.p1 = Enter your postcode to find your address automatically.
 indWhatIsYourPostcode.checkYourAnswersLabel = What is your postcode?
 indWhatIsYourPostcode.button = Find address
 indWhatIsYourPostcode.manualLink = Or enter the address manually
-indWhatIsYourPostcode.error.required = Enter your postcode
-indWhatIsYourPostcode.error.length = Your postcode must be 8 characters or less
-indWhatIsYourPostcode.error.invalid = Enter a real postcode or enter the address manually
-indWhatIsYourPostcode.error.notFound = Address not found - enter a different postcode or enter the address manually
+indWhatIsYourPostcode.error.required = 	Enter a postcode, like AA1 1AA
```

Comment:
There's an extra space in front of ' Enter a postcode, like AA1 1AA'. Is that intended?